reana_commons/config.py

```diff
@@ -224,6 +225,35 @@
 )
 """Name of the shared CEPHFS PVC which will be used by all REANA jobs."""
 
+REANA_JOB_HOSTPATH_MOUNTS = json.loads(os.getenv("REANA_JOB_HOSTPATH_MOUNTS", "[]"))
+"""List of dictionaries composed of name, hostPath and mountPath.
+
+- ``name``: name of the mount.
+- ``hotsPath``: path to the directory in the Kubernetes nodes to be mounted in the jobs.
+- ``mountPath``: path to mount the Kubernetes nodes directory to inside the job containers.
+  This is optional, by default the same path as the hostPath will be used
+
+This configuration should be used only when one knows for sure that the
+specified locations exist in all the cluster nodes. For example, in Minikube:
+
+.. code-block::
+
+    $ minikube mount /usr/local/share/mydata:/mydata
+
+And pass the following configuration:
+
+.. code-block::
+
+    REANA_JOB_HOSTPATH_MOUNTS = [
+        {"name": "mydata",
+         "hostPath": "/var/mydata",
```

Comment:
Note that the example has hostPath and mountPath inversed, because minikube mounts stuff into `/mydata`, not to `/var/mydata`...

Perhaps use the following, which may be closer to reality (i.e. people having inputs in their private folder).

- laptop: $HOME/mydata
- `minikube mount $HOME/mydata:/usr/local/share/mydata`
- hostpath: /usr/local/share/mydata
- mountpath: /mydata
- jobs: `ls -l /mydata`

Reply:
Thanks, fixed :)